lib/movie_card/movie_card.dart

```diff
@@ -13,75 +13,101 @@ class MovieCard extends StatelessWidget {
+              children: [
+                Stack(
                   children: [
-                    Padding(
-                      padding: const EdgeInsets.only(bottom: 5),
-                      child: Text(
-                        '${movie.topPosition}',
-                        style: const TextStyle(
-                          color: Colors.white,
-                          fontSize: 16,
+                    Image.asset(movie.poster),
+                    const AddToWatchlistTab(),
+                  ],
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(10, 10, 0, 5),
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(bottom: 5),
+                        child: Text(
+                          '${movie.topPosition}',
+                          style: const TextStyle(
+                            color: Colors.white,
+                            fontSize: 16,
+                          ),
                         ),
                       ),
-                    ),
-                    Padding(
-                      padding: const EdgeInsets.only(bottom: 5),
-                      child: Text(
-                        movie.title,
-                        style: const TextStyle(
-                          color: Colors.white,
-                          fontSize: 16,
+                      Padding(
+                        padding: const EdgeInsets.only(bottom: 5),
+                        child: Text(
+                          movie.title,
+                          style: const TextStyle(
+                            color: Colors.white,
+                            fontSize: 16,
+                          ),
                         ),
                       ),
-                    ),
-                    Row(
-                      crossAxisAlignment: CrossAxisAlignment.end,
-                      children: [
-                        const Padding(
-                          padding: EdgeInsets.only(right: 3),
-                          child: Icon(IconData(0xf01d3, fontFamily: 'MaterialIcons'), color:Color.fromARGB(255, 230, 199, 25), size: 21),
-                        ),
-                        Padding(
-                          padding: const EdgeInsets.only(right: 15),
-                          child: Text('${movie.rating}', style: const TextStyle(color: Colors.white, fontSize: 16,),),
+                      Row(
+                        crossAxisAlignment: CrossAxisAlignment.end,
+                        children: [
+                          const Padding(
+                            padding: EdgeInsets.only(right: 3),
+                            child: Icon(
+                              IconData(0xf01d3, fontFamily: 'MaterialIcons'),
```

Comment:
Replace with proper `Icon.xxx`.
You can find the list of material icons here https://fonts.google.com/icons?icon.query=bookmark.